packages/vite/src/node/plugins/css.ts

```diff
@@ -378,8 +377,8 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
 
   // when there are multiple rollup outputs and extracting CSS, only emit once,
   // since output formats have no effect on the generated CSS.
-  let outputToExtractedCSSMap: Map<NormalizedOutputOptions, string>
   let hasEmitted = false
+  let chunkCSSMap: Map<string, string>
```

Comment:
I guess we have to make this map `Map<NormalizedOutputOptions, Map<string, string>>`.
IIUC this `NormalizedOutputOptions` is necessary because `RenderChunk` hook will be called multiple times for the same chunk if multiple formats is used (https://github.com/vitejs/vite/commit/6bce1081991501f3779bff1a81e5dd1e63e5d38e).

Reply:
However,  it seems unnecessary because the map will only be used once.

>   // when there are multiple rollup outputs and extracting CSS, only emit once,
>   // since output formats have no effect on the generated CSS.

And according to 6bce108, "output formats have no effect on the generated CSS". So even if we make it `Map<NormalizedOutputOptions, Map<string, string>>`, the values of the map should be identical.

I think Evan originally created this nested map because he was appending CSS directly during `renderChunk`:

```
        outputToExtractedCSSMap.set(opts, extractedCss + chunkCSS)
```

where the CSS of a chunk can possibly be appended more than once. But since we are "setting" not "appending" now, I think it's safe.

Reply:
Ah, you're right. I forgot that we're setting it😅.